openssl-sys/src/x509.rs

```diff
@@ -225,6 +225,8 @@ extern "C" {
     pub fn X509_get_pubkey(x: *mut X509) -> *mut EVP_PKEY;
 
     pub fn X509_set_version(x: *mut X509, version: c_long) -> c_int;
+    #[cfg(ossl110)]
```

Comment:
It is unclear to me how comes that `X509_set_version` and `X509Builder::set_version` methods do not need `#[cfg(ossl110)]` constraint declared. OpenSSL docs for [v1.0.2](https://www.openssl.org/docs/man1.0.2/man3/) do not list a possibility to set X509 certificate version.

Reply:
OpenSSL's man pages do not cover every single function the library exposes.